res/layout/microphone_recorder_view.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    tools:showIn="@layout/conversation_input_panel">
+
+    <ImageButton
+        android:id="@+id/quick_audio_toggle"
+        android:layout_width="wrap_content"
+        android:layout_height="match_parent"
+        android:layout_gravity="center_vertical"
+        android:paddingLeft="11dp"
+        android:paddingRight="11dp"
+        android:background="@null"
+        android:contentDescription="@string/audio"
+        android:src="?quick_mic_icon" />
+
+    <LinearLayout
+        android:id="@+id/lock_drop_target"
+        android:layout_width="40dp"
+        android:layout_height="64dp"
+        android:layout_gravity="center"
+        android:layout_marginLeft="-9dp"
+        android:layout_marginStart="-9dp"
```

Comment:
This corrects a small shift to the right of the lock drop target.

I have no idea why the shift was there in the first place, though.